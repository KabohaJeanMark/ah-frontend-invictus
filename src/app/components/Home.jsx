import React, { Component } from 'react';
import NavbarComponent from '../components/Navbar';
import Articles from '../containers/articles/ArticlesContainer';

class Home extends Component {
  render () {
    return (
      <React.Fragment>
        <NavbarComponent />
        <Articles />
      </React.Fragment>
    );
  }
}

export default Home;
