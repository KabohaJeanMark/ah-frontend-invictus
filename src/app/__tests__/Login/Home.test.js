import React from 'react';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';
import Home from '../../components/Home';

describe('<Home />', () => {
  let wrapper = shallow(<Home />);

  it('matches snapshot', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
