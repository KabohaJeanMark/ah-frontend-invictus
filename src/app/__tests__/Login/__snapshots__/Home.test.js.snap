// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> matches snapshot 1`] = `
<Fragment>
  <Connect(NavbarComponent) />
  <Connect(Articles) />
</Fragment>
`;
